src/BitOasis/Coin/DefaultCryptocurrencyFactory.php

```diff
@@ -125,6 +125,16 @@ protected function addDefaultCryptocurrencies() {
 		$this->cryptocurrencies[Cryptocurrency::VRA] = new Cryptocurrency(Cryptocurrency::VRA, 18, 'Verasity');
 		$this->cryptocurrencies[Cryptocurrency::UTK] = new Cryptocurrency(Cryptocurrency::UTK, 18, 'Utrust');
 		$this->cryptocurrencies[Cryptocurrency::SGB] = new Cryptocurrency(Cryptocurrency::SGB, 18, 'Songbird');
+		$this->cryptocurrencies[Cryptocurrency::BLUR] = new Cryptocurrency(Cryptocurrency::BLUR, 18, 'Blur');
+		$this->cryptocurrencies[Cryptocurrency::OP] = new Cryptocurrency(Cryptocurrency::OP, 18, 'Optimism');
+		$this->cryptocurrencies[Cryptocurrency::QRDO] = new Cryptocurrency(Cryptocurrency::QRDO, 18, 'QREDO');
+		$this->cryptocurrencies[Cryptocurrency::BOSON] = new Cryptocurrency(Cryptocurrency::BOSON, 18, 'BOSON');
+		$this->cryptocurrencies[Cryptocurrency::FLOKI] = new Cryptocurrency(Cryptocurrency::FLOKI, 18, 'FLOKI');
+		$this->cryptocurrencies[Cryptocurrency::WILD] = new Cryptocurrency(Cryptocurrency::WILD, 18, 'Wilder World');
+		$this->cryptocurrencies[Cryptocurrency::SUI] = new Cryptocurrency(Cryptocurrency::SUI, 9, 'Sui');
+		$this->cryptocurrencies[Cryptocurrency::SEI] = new Cryptocurrency(Cryptocurrency::SEI, 6, 'Sei Network');
```

Comment:
Is this correct 6? not 18

Reply:
Where you got info about 18 for Sei?

Reply:
Not sure if this trusted but 

https://etherscan.io/token/0x9c1cb740f3b631ed53600058ae5b2f83e15d9fbf
https://ethplorer.io/address/0x9c1cb740f3b631ed53600058ae5b2f83e15d9fbf#

Reply:
Its not ERC20 though